File: 형우/배열과문자열/URL화.java
import java.util.Arrays;
import java.util.stream.Collectors;

public class URL화 {
    private static final String blank = "%20";

    // URL 화
    public static void main(String[] args) {
        String s = "Mr John Smith";
        System.out.println(makeURL(s));
    }

    static public String makeURL(String s) {
        String[] splitString = s.split(" ");
        return Arrays.stream(splitString).collect(Collectors.joining(blank));
    }
}

File: 영동/배열과문자열/URL화.py
# 문자열에 들어있는 모든 공백을 %20로 바꿔주는 메서드
# 유니코드에서 공백 = %20

def convertSpaceToUnicode(string, length):
    sentence = ''
    for i in string:
        if i == ' ':
            sentence += '%20'
        else:
            sentence += i
    return sentence

print(convertSpaceToUnicode('Mr John Smith', 13))

